.circleci/config.yml

```diff
@@ -10,6 +10,9 @@ jobs:
       - image: cimg/openjdk:11.0
     steps:
       - checkout
+      - run:
+          name: Install dependencies
+          command: sudo apt update && xargs -a app/dependencies.txt sudo apt install
```

Comment:
Let's just assume that NASM is already installed (and write in README about this). Also, not everyone uses `apt`...